pkg/proxy/service.go

```diff
@@ -139,6 +139,24 @@ func (info *BaseServiceInfo) HintsAnnotation() string {
 	return info.hintsAnnotation
 }
 
+// ExternallyAccessible is part of ServicePort interface.
+func (info *BaseServiceInfo) ExternallyAccessible() bool {
+	return info.nodePort != 0 || len(info.loadBalancerStatus.Ingress) != 0 || len(info.externalIPs) != 0
```

Comment:
Should this include node ports?  Especially is fame-node nodeport is == LBVIP semantics?  Mayber a followup

Reply:
Yes, it should (and does) include nodeports. The function is answering the question "Can the service ever receive traffic for which we would need to apply `externalTrafficPolicy`?".

Reply:
Sorry I totally missed the `nodePort` in there.  wow.